dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/confirmemail/ConfirmEmailUtil.java

```diff
@@ -53,4 +54,35 @@ public static String friendlyExpirationTime(long expirationLong) {
         }
     }
 
+    public static String friendlyExpirationTime(long expirationLong, Locale language) {
```

Comment:
Is method without Locale working correctly (we don't want locale in every instance)?.

Reply:
They both work correctly. The distinction is: email notifications are specific to language specified by user in his account, while notifications in the app itself are based on currently selected language.

Hence we need both methods: the one with locale is for emails, the one without is for app notifications.